pkg/domain/entities/pods.go

```diff
@@ -152,24 +152,24 @@ type ContainerCreateOptions struct {
 	CapDrop           []string
 	CgroupNS          string
 	CGroupsMode       string
-	CGroupParent      string
+	CGroupParent      string `json:"cgroup_parent,omitempty"`
 	CIDFile           string
-	ConmonPIDFile     string
+	ConmonPIDFile     string `json:"infra_conmon_pidfile,omitempty"`
 	CPUPeriod         uint64
 	CPUQuota          int64
 	CPURTPeriod       uint64
 	CPURTRuntime      int64
 	CPUShares         uint64
-	CPUS              float64
-	CPUSetCPUs        string
+	CPUS              float64 `json:"cpus,omitempty"`
+	CPUSetCPUs        string  `json:"cpuset_cpus,omitempty"`
 	CPUSetMems        string
 	Devices           []string
 	DeviceCGroupRule  []string
 	DeviceReadBPs     []string
 	DeviceReadIOPs    []string
 	DeviceWriteBPs    []string
 	DeviceWriteIOPs   []string
-	Entrypoint        *string
+	Entrypoint        *string `json:"infra_command,omitempty"`
```

Comment:
is infra_command right here?

Reply:
I can make it more general, podcreateoptions and containercreateoptions just need to match